# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/VertexBuffer.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/renderer/d3d/VertexBuffer.h
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/VertexBuffer.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/VertexBuffer.h	Sat Sep 07 13:38:36 2019 +0000
@@ -13,6 +13,7 @@
 #include "common/PackedEnums.h"
 #include "common/angleutils.h"
 #include "libANGLE/Error.h"
+#include "libANGLE/renderer/Format.h"
 
 #include <GLES2/gl2.h>
 
@@ -172,11 +173,8 @@
         void set(const gl::VertexAttribute &attrib, const gl::VertexBinding &binding);
 
       private:
-        gl::VertexAttribType type;
-        GLuint size;
+        angle::FormatID formatID;
         GLuint stride;
-        bool normalized;
-        bool pureInteger;
         size_t offset;
     };
 